packages/graphql-transformers-e2e-tests/src/__tests__/DynamoDBModelTransformer.e2e.test.ts

```diff
@@ -11,7 +11,7 @@ import { Output } from 'aws-sdk/clients/cloudformation'
 import { GraphQLClient } from '../GraphQLClient'
 import * as moment from 'moment';
 
-jest.setTimeout(200000);
+jest.setTimeout(2000000);
```

Comment:
nit: Can we move the timeout to a constant or use  all of these using `setupFiles` in config

Reply:
Its okay with me. Could use some guidance on how to do that.

Reply:
In `packages/graphql-transformers-e2e-tests/package.json`s `jest` section you could add [`setupfiles`](https://github.com/facebook/jest/blob/master/docs/Configuration.md#setupfiles-array) and there you could add `jest.setTimeout`. This would set the time out to the specified time in each and every test.

Reply:
Can this be a follow up PR?

Reply:
This is blocking 2 other PRs that solve customer bugs

Reply:
I already have approved the PR.

Reply:
Thanks! I'll make a note to follow up.